### PR TITLE
back office: add top navigation

### PR DESCRIPTION
**Pull request by NRodriguezcuellar**
closes https://github.com/inveniosoftware/invenio-administration/issues/2
needs https://github.com/inveniosoftware/invenio-app-rdm/pull/1823

Screenshot from 2022-08-04 11-29-15

Screenshot from 2022-08-04 11-30-40
Screenshot from 2022-08-04 11-29-26

### CODE REVIEW

**Review by kpsherva — APPROVED** [review]

File: invenio_administration/assets/semantic-ui/js/invenio_administration/theme.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import $ from "jquery";
+
+const menuId = "#invenio-admin-top-nav"
+
+/* Expand and collapse navbar  */
+const toggleIcon = $("#rdm-burger-menu-icon");
```
Comment (kpsherva) [inline invenio_administration/assets/semantic-ui/js/invenio_administration/theme.js:6]: I think we should refactor it and move to invenio-theme, but maybe in a separate PR

File: invenio_administration/templates/semantic-ui/invenio_administration/base.html [hunk 2]
```diff
@@ -6,15 +6,15 @@
 #}
 
 <!DOCTYPE html>
-{%- extends config.BASE_TEMPLATE %}
+{%- extends "invenio_theme/page.html" %}
```
Comment (kpsherva) [inline invenio_administration/templates/semantic-ui/invenio_administration/base.html:9]: how about:
```suggestion
{%- extends config.ADMINISTRATION_THEME_BASE_TEMPLATE %}
```

File: invenio_administration/templates/semantic-ui/invenio_administration/base.html [hunk 3]
```diff
@@ -6,15 +6,15 @@
...
 
 {% block bypasslinks %}
 {% endblock %}
 
 {% block body_inner %}
-  <div class="ui grid rel-pl-1 rel-pr-1">
+  <div class="ui grid rel-pl-1 rel-pr-1 invenio-administration">
     {% block admin_side_column %}
-      <div class="three wide column" style="border: 1px solid grey; min-height: 100vh;">
+      <div class="three wide column" style="border: 1px solid transparent; min-height: 100vh;">
```
Comment (kpsherva) [inline invenio_administration/templates/semantic-ui/invenio_administration/base.html:17]: please remove the inline styling, everything should go to less theme files

File: invenio_administration/assets/semantic-ui/js/invenio_administration/theme.js (hunk 1 above)
Comment (NRodriguezcuellar) [reply invenio_administration/assets/semantic-ui/js/invenio_administration/theme.js:6]: yes i agree!

**Pull request merged**